src/polyfills/includes.polyfill.js

```diff
@@ -0,0 +1,32 @@
+if (!String.prototype.includes) {
+  String.prototype.includes = function(search, start) {
+    "use strict";
+    if (typeof start !== "number") {
+      start = 0;
+    }
+
+    if (start + search.length > this.length) {
+      return false;
+    } else {
+      return this.indexOf(search, start) !== -1;
+    }
+  };
+}
+
+if (!Element.prototype.matches) {
```

Comment:
this should be in it's own file, so that it can be imported only if needed to keep files sizes down.

Reply:
if that is what we are going to do instead of importing all polyfills then we need to name the files based on what they are polyfilling for easier and quicker reference.

Reply:
isn't that what we are doing. the first polyfill in this file is "includes"?

Reply:
appears that way

Reply:
this is used by the closest polyfill so I dont know if you want me to break this out into a matches polyfill and import in the closest or just leave in closest.

Reply:
ok, i didn't knwo that, leaving it in there is fine by me then if you are cool with it.